deploy/helm/sumologic/README.md

```diff
@@ -283,19 +283,19 @@ The following table lists the configurable parameters of the Sumo Logic chart an
 | `opentelemetry-operator.enabled`                                                      | Flag to control deploying OpenTelemetry Operator Helm sub-chart.                                                                                                                                                                                                                                           | `true`                                                                                                                                                                               |
 | `opentelemetry-operator.createDefaultInstrumentation`                                 | Flag to control creation of default Instrumentation object                                                                                                                                                                                                                                                 | `false`                                                                                                                                                                              |
 | `opentelemetry-operator.instrumentation.dotnet.extraEnvVars`                          | Additional environment variables for OpenTelemetry Operator Instrumentation Resource.                                                                                                                                                                                                                      | `{}`                                                                                                                                                                                 |
-| `opentelemetry-operator.instrumentation.dotnet.image`                                 | OpenTelemetry Operator DotNet Instrumentation image.                                                                                                                                                                                                                                                       | `ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-dotnet:0.7.0`                                                                                                     |
+| `opentelemetry-operator.instrumentation.dotnet.image`                                 | OpenTelemetry Operator DotNet Instrumentation image.                                                                                                                                                                                                                                                       | `ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-dotnet:1.4.0`                                                                                                     |
```

Comment:
This is a major breaking change due to changes to HTTP semantic conventions. Shouldn't we hold back with it? At the very least, we should add a warning to the changelog about this.

Reply:
Good catch, I've mixed core vs instr versioning... I will revert to the previous version used. Lack of support on our side for new sem conv will be growing pain soon.

Reply:
I'm pretty sure 1.2.0 is the latest version without the semconv changes, so we can upgrade to that.